src/main/java/Duke.java

```diff
@@ -1,10 +1,111 @@
+import task.*;
+
+import java.io.*;
+import java.text.ParseException;
+
 public class Duke {
+
+    private static final String FILEPATH = "duke.txt";
```

Comment:
Shoud FILE_PATH be a better choice here?

Reply:
I agree with using an underscore to separate the words here. Gives better readability